src/core/jsproxy.c

```diff
@@ -57,7 +57,7 @@
 #define IS_BUFFER          (1 << 8)
 #define IS_CALLABLE        (1 << 9)
 #define IS_ARRAY           (1 << 10)
-#define IS_NODE_LIST       (1 << 11)
+#define IS_ARRAY_LIKE      (1 << 11)
```

Comment:
Maybe it would be nice to add a comment about what the ARRAY_LIKE is here as well.

```
 * array-like objects like HTMLCollection or
 * NodeList, controlled by IS_ARRAY_LIKE
```